BTrees/tests/test_OOBTree.py

```diff
@@ -126,6 +126,12 @@ class C(object):
 
         self.assertRaises(TypeError, lambda : t.__setitem__(C(), 1))
 
+        try:
+            t[C()] = 1
+            self.fail("Should raise TypeError")
+        except TypeError as e:
+            self.assertEqual(e.args[0], "Object has default comparison")
+
```

Comment:
ISTM this would be clearer / more idiomatic:

```python
        with self.assertRaises(TypeError) as raises:
            t[C()] = 1

        self.assertEquals(raiess.exception.args[0], "Object has default comparison")
```

See: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises